===================================================================
RCS file: /usr/local/www/cvsroot/FreeBSD/src/sys/i386/i386/local_apic.c,v
retrieving revision 1.26
retrieving revision 1.27
diff -u -p -r1.26 -r1.27
--- src/sys/i386/i386/local_apic.c	2006/02/28 22:24:55	1.26
+++ src/sys/i386/i386/local_apic.c	2006/03/20 19:39:08	1.27
@@ -208,7 +208,7 @@ lapic_init(uintptr_t addr)
 	/* Map the local APIC and setup the spurious interrupt handler. */
 	KASSERT(trunc_page(addr) == addr,
 	    ("local APIC not aligned on a page boundary"));
-	lapic = (lapic_t *)pmap_mapdev(addr, sizeof(lapic_t));
+	lapic = pmap_mapdev(addr, sizeof(lapic_t));
 	setidt(APIC_SPURIOUS_INT, IDTVEC(spuriousint), SDT_SYS386IGT, SEL_KPL,
 	    GSEL(GCODE_SEL, SEL_KPL));
 

